modules/swagger-codegen/src/main/resources/apex/Swagger.cls

```diff
@@ -176,11 +176,23 @@ public class Swagger {
             if (isFailure) {
                 throw new ApiException(code, res.getStatus(), getHeaders(res), res.getBody());
             } else if (returnType != null) {
-                return toReturnValue(res.getBody(), returnType, res.getHeader('Content-Type'));
+                return toReturnValue(res.getBody(), returnType, getContentType(res));
             }
             return null;
         }
 
+        private String getContentType(HttpResponse res) {
+            List<String> headerKeys = res.getHeaderKeys();
+
+            for (String headerKey: headerKeys) {
+                if (headerKey.equalsIgnoreCase('content-type')) {
```

Comment:
This is fine but might consider passing in the desired header so the method can be reused.